cmd/minikube/cmd/extract.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
It seems like overkill to include this functionality with the end-user utility. What's the advantage here?

Reply:
It seemed like the easiest way to get extract lit of `hack` and into its own package so I can properly test it. Is there an easier way I'm missing?

Reply:
Nevermind, I moved it out of minikube/cmd. This is way easier.